src/components/site-header/styles/_Header.scss

```diff
@@ -11,6 +11,16 @@
   a.logo img {
       max-height: 50px;
   }
+
+  .getsmarter-logo {
+    background: transparent;
+    background-color: #082135;
```

Comment:
nit: Maybe define this hex value as a SCSS variable? Also, is this `background-color` just overriding the `background` property above?

Reply:
oops yeah.